### PR TITLE
China Lake hotfix

### PR DESCRIPTION
**Pull request by genessee596**
## About The Pull Request

image

It worked before I added preloaded. I didn't test my code again after I added this - that's my bad, heehoo. This just makes it so the lake will spawn with a single frag shot, similar to how voidsuits spawn with a pair of magboots.

## Why It's Good For The Game

Things working is very good.

## Changelog
:cl:
fix: China Lake works for real this time. I promise.
/:cl:

### CODE REVIEW

**Review by TheShown911 — APPROVED** [review]

**Comment by genessee596** [comment]
bugfix with maintainer approval, merging

File: code/modules/projectiles/guns/launcher/projectile_grenade_launcher.dm [hunk 1]
```diff
@@ -20,9 +20,6 @@
     max_shells = 3
     recoil_buildup = 20
     twohanded = TRUE
-    bad_type = /obj/item/weapon/gun/projectile/shotgun/pump/china
-
-/obj/item/weapon/gun/projectile/shotgun/pump/china/preloaded
     spawn_tags = SPAWN_TAG_GUN_PROJECTILE
```
Comment (Evankhell561) [inline code/modules/projectiles/guns/launcher/projectile_grenade_launcher.dm:23]: I keep wondering why this change in the label of this weapon. not a shotgun?
